rest-api/src/main/java/com/cloudbees/workflow/rest/external/PendingInputActionsExt.java

```diff
@@ -111,6 +114,7 @@ public static PendingInputActionsExt create(InputStepExecution inputStepExecutio
         String inputId = inputStepExecution.getId();
         inputActionExt.setId(inputId);
         inputActionExt.setProceedText(inputStepExecution.getInput().getOk());
+        inputActionExt.setAbortText(inputStepExecution.getInput().getAbort());
```

Comment:
I think you meant:

```suggestion
        inputActionExt.setAbortText(inputStepExecution.getInput().getAbortText());
```

getAbort() is not a method